src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -60,13 +61,20 @@ public DependencyResolver(GitCommitProvider gitCommitProvider, Lazy<XrefMap> xre
             }
 
             // need to url decode uid from input content
-            var (xrefSpec, referencedFile) = _xrefMap.Value.Resolve(HttpUtility.UrlDecode(uid), moniker);
+            var (xrefSpec, referencedFile, referencedCallStack) = _xrefMap.Value.Resolve(HttpUtility.UrlDecode(uid), relativeTo, moniker);
```

Comment:
>Resolve [](start = 81, length = 7)

1. try add ref to callstack here and handle conflicts
2. then pass to resolve
3. pop current ref

Reply:
The problem here is that, while we build xref map we will create a lazy value of `xrefSpec`, in this callback we will call `ResolveXref`, we do not have `callStack` ready yet. So I prepared `callStack` along with build file, and return another `callStack` when uid resolved.